attitude_fw/ecl_roll_controller.cpp

```diff
@@ -136,7 +129,12 @@ float ECL_RollController::control_euler_rate(const struct ECL_ControlData &ctl_d
 	/* Transform setpoint to body angular rates (jacobian) */
 	_bodyrate_setpoint = ctl_data.roll_rate_setpoint - sinf(ctl_data.pitch) * ctl_data.yaw_rate_setpoint;
 
+	/* limit the rate */
+	if (_max_rate > 0.01f) {
+		_bodyrate_setpoint = (_bodyrate_setpoint > _max_rate) ? _max_rate : _bodyrate_setpoint;
```

Comment:
You might want to use math::constrain everywhere instead. https://github.com/PX4/ecl/blob/master/attitude_fw/ecl_roll_controller.cpp#L88

It's easier to read and harder to screw up.

Reply:
sure. will do.